fapolicy_analyzer/ui/policy_rules_admin_page.py

```diff
@@ -385,6 +402,7 @@ def __populate_objects(self):
             or self.__selection_state["group"] is not None
         ):
             last_subject = self.__selection_state["subjects"][-1]
+            self.when_none = sum([True for e in self.__log.by_subject(last_subject) if e.when() is None])
```

Comment:
Minor suggestion, but you can simplify this statement with something like this:
```suggestion
            self.when_none = any(e is None for e in self.__log.by_subject(last_subject))
```

Reply:
Thanks Shawn! Should be good to go.